src/avram/tasks/db/base_task.cr

```diff
@@ -0,0 +1,9 @@
+abstract class BaseTask < LuckyCli::Task
```

Comment:
This abstract class was pulled out so that we could test and use tasks without accidentally running into the `exit` code.

Reply:
oh nice! Good call.